core/src/main/java/io/neow3j/protocol/core/response/ContractManifest.java

```diff
@@ -92,6 +93,24 @@ public List<ContractGroup> getGroups() {
         return groups;
     }
 
+    @JsonIgnore
+    public ContractGroup getFirstGroup() {
+        if (groups.size() == 0) {
```

Comment:
So, I assume here that `groups` is never null because of `@JsonSetter(nulls = Nulls.AS_EMPTY)`, right?

Reply:
This question also applies to all other attributes which has similar behavior.

Reply:
Hmm, actually, it seems the JsonSetter only initializes it as an empty array its value is `null`, but not if the value was not present during deserialization.

Reply:
In this case, I suggest adding some tests to verify that.